doc/neps/nep-0029-deprecation_policy.rst

```diff
@@ -114,7 +114,12 @@ Jul 26, 2021 3.7+   1.18+
 Dec 22, 2021 3.7+   1.19+
 Dec 26, 2021 3.8+   1.19+
 Jun 21, 2022 3.8+   1.20+
-Apr 14, 2023 3.9+   1.20+
+Jan 31, 2023 3.8+   1.21+
+Apr 14, 2023 3.9+   1.21+
+Jun 23, 2023 3.9+   1.22+
+Jan 01, 2024 3.9+   1.23+
+Apr 05, 2024 3.10+  1.23+
+Apr 04, 2025 3.11+  1.23+
```

Comment:
I don't understand this table. Could we improve the top line? According to this, today we are supporting NumPy 1.20, or we are recommending other projects use NumPy 1.20+? The table is unclear.

Reply:
Yeah, I didn't change the table. If we want to, then that probably needs a bit more discussion.

I'm reading the `1.23+` as "all available numpy versions `1.x` where `x>=23`. However, the table is not exactly true because once we add the as-yet unreleased numpy releases to the list, the last line will probably end up `Apr 04, 2025 3.11+  1.25+`. This was also discussed in https://github.com/numpy/numpy/pull/17337.